types/react-map-gl/index.d.ts

```diff
@@ -7,6 +7,8 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.0
 
+/// <reference lib='dom' />
```

Comment:
Is this needed? It looks like the existing definitions are fine without this reference?

Reply:
I had an issue with [next.js](https://nextjs.org) where tsc couldn't find definitions for some types used in this file, such as `CanvasRenderingContext2D`, `WebGLContextEvent`, `ImageData`. Adding this reference to the DOM lib fixed the issue.